Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/HolographicCamera/CalibrationDataProvider.cs

```diff
@@ -3,6 +3,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.IO;
+using System.Runtime.InteropServices.WindowsRuntime;
 using UnityEngine;
```

Comment:
Does WindowsRuntime resolve for non UNITY_WSA?

Reply:
I had issues with some Interop service APIs not working on IL2CPP with unity wsa too. I'm a bit weary

Reply:
This particular method seems to be working on IL2CPP (x86). I've moved the using into the UNITY_WSA-specific block.